packages/client/hmi-client/src/page/project/components/tera-save-asset-modal.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+	<Teleport to="body">
+		<tera-modal
+			v-if="isVisible"
+			class="save-as-dialog"
+			@modal-mask-clicked="closeModal"
+			@on-modal-open="initializeAsset"
+			@modal-enter-press="save"
+		>
+			<template #header>
+				<h4>{{ title }}</h4>
+			</template>
+			<template #default>
+				<form @submit.prevent>
+					<label for="new-name">What would you like to call it?</label>
+					<InputText
+						id="new-name"
+						type="text"
+						v-model="newName"
+						placeholder="Enter a unique name"
+					/>
+					<slot name="extra-input-fields" />
```

Comment:
If nobody is using it, let's remove it. We will figure about future uses later on.
Adding a `description` test area would be great here, and making it not mandatory.